wallet/src/wallet.rs

```diff
@@ -98,7 +98,6 @@ pub trait CapeWalletExt<'a, Backend: CapeWalletBackend<'a> + Sync + 'a> {
     /// does, use [CapeWalletExt::sponsor].
     async fn build_sponsor(
```

Comment:
I think we should remove "added to the asset library" from the comment.

Reply:
Oops good catch